package/doc/sphinx/source/documentation_pages/coordinates/pickle_universe.rst

```diff
@@ -0,0 +1,117 @@
+.. Contains the formatted docstrings for the serialization of universe located 
+.. mainly in 'MDAnalysis/libs/pickle_file_io.py'
+.. _serialization:
+
+*********************************************************
+Serialization of Universes
```

Comment:
I asked for the whole doc to be put under coordinates because ultimately this is mainly about serializing the readers and what developers need to know. Or at least, that's currently my main concern.

I agree with @kain88-de that a split into two docs would be good:
1. developers: *what you need to do to get a reader to serialize*
2. user facing document on *how to use serialization*

Reply:
@yuxuanzhuang Make this document the one about **Serialization of Coordinate Readers**
- change the heading
- focus text on what developers need to know